### PR TITLE
feat(env-checker): [part-2] Implement dialog UI in CLI env checker

### PR DESCRIPTION
**Pull request by a1exwang**
- Implement dialog related features in CLI env checker in the adatper

image

https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/10338893

### CODE REVIEW

File: packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts [hunk 1]
```diff
@@ -4,6 +4,9 @@ import { IDepsAdapter, IDepsTelemetry } from "./checker";
 import { cliEnvCheckerLogger as logger } from "./cliLogger";
 import { cliTelemetry } from "./cliTelemetry";
 import open from "open";
+import DialogManagerInstance from "../../../userInterface";
+// import { CliTelemetry } from "./telemetry/cliTelemetry";
```
Comment (meifans) [inline packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts]: remove unused  code

**Review by meifans — APPROVED** [review]

File: packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts (hunk 1 above)
Comment (a1exwang) [reply packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts]: already removed

File: packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts [hunk 2]
```diff
@@ -3,7 +3,8 @@ import { DepsCheckerEvent, Messages } from "./common";
 import { IDepsAdapter, IDepsTelemetry } from "./checker";
 import { cliEnvCheckerLogger as logger } from "./cliLogger";
 import { cliTelemetry } from "./cliTelemetry";
-import open from "open";
+import DialogManagerInstance from "../../../userInterface";
```
Comment (kuojianlu) [inline packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts:6]: Seems not used.

Comment (a1exwang) [reply packages/cli/src/cmds/preview/depsChecker/cliAdapter.ts:6]: Already merged.... Will remove in the next PR.

**Pull request merged**